### PR TITLE
util for getting top style value of an item when another item is clic…

### PR DESCRIPTION
**Pull request by pchatterton**
…ked on

### CODE REVIEW

**Comment by mndvns** [comment]
is this actually used anywhere other than lists?

**Comment by pchatterton** [comment]
It'll be used in poll-taker-ui also. I've been working on making the breakdown look like a modal that will have the same behavior as the reader's likst top position.

**Comment by mndvns** [comment]
ah, right. i wish i could help you out with that one, man. godspeed!

**Comment by mndvns** [comment]
feel free to PM or text me or something if you need help @pchatterton

File: utils/get-top-position.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*
+ * offset - distance from clicked element and container with relative position
+ * top - distance from clicked element to top of window
+ * itemHeight - height of the item that will be positioned
+ * containerHeight - height of the container with relative position
+ * adjustment - adjust offset of clicked element (used for some cases)
+*/
+
+export default function(offset, top, itemHeight, containerHeight, adjustment) {
```
Comment (jakerichan) [inline utils/get-top-position.js]: what if we used a single `options` argument so if we ever want to set things like padding through the function, it's easy to update.

**Comment by pchatterton** [comment]
thanks @mndvns! I appreciate it. Even if you have feedback on things you see just let me know.

**Comment by pchatterton** [comment]
@mndvns Just saw your comment. I refactored most of it earlier today, including those if statements.

**Comment by bbassett** [comment]
:+1:

**Pull request merged**